clang/docs/analyzer/checkers.rst

```diff
@@ -588,8 +588,8 @@ Warns when a null pointer is passed to a pointer which has a _Nonnull type.
 
 .. _nullability-NullReturnedFromNonnull:
 
-nullability.NullReturnedFromNonnull (ObjC)
-""""""""""""""""""""""""""""""""""""""""""
+nullability.NullReturnedFromNonnull
+"""""""""""""""""""""""""""""""""""
```

Comment:
```suggestion
nullability.NullReturnedFromNonnull (C, C++, ObjC)
""""""""""""""""""""""""""""""""""""""""""""""""""
```
This is how other checkers are marked. (Do I understand correctly that `__attribute__((returns_nonnull))` is relevant both under C and C++?)

Reply:
Yes, it is relevant for both. Thanks for the fix! Applied